### PR TITLE
VITA eligibility

### LINKED ISSUE #4720: VITA eligibility
https://www.irs.gov/individuals/free-tax-return-preparation-for-qualifying-taxpayers

> VITA sites offer free tax help to people who need assistance in preparing their own tax returns, including:
> 
> People who generally make $64,000 or less
> Persons with disabilities; and
> Limited English-speaking taxpayers
> In addition to VITA, the TCE program offers free tax help, particularly for those who are 60 years of age and older, specializing in questions about pensions and retirement-related issues unique to seniors.

I don't think there's explicit legal/regulatory backing for these. SingleStop models it.

### PR DESCRIPTION
**Pull request by shuyu7zhan**
Fixes #4720

copilot:all

### CODE REVIEW

File: policyengine_us/parameters/gov/irs/vita/eligibility/income_limit.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description: IRS's Volunteer Income Tax Assistance (VITA) sites offer free tax help to people who make income no more than this amount.
+values:
+  2021-01-01: 64_000
+metadata:
+  unit: currency-USD
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/irs/vita/eligibility/income_limit.yaml:5]:
```suggestion
  unit: currency-USD
  period: year
```

File: policyengine_us/parameters/gov/irs/vita/eligibility/income_limit.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  label: VITA Program Income Limit
+  reference:
+    - title: Free tax return preparation for qualifying taxpayers
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/irs/vita/eligibility/income_limit.yaml:8]: we should see if we can find more official references - especially for start dates and values in 2021 / 2022 / 2023

File: policyengine_us/tests/policy/baseline/gov/irs/vita/vita_eligibility.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+- name: IRS VITA Program Eligible.
+  period: 2023
+  input:
+    irs_gross_income: 60_000
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/irs/vita/vita_eligibility.yaml:4]: add more tests

File: policyengine_us/parameters/gov/irs/vita/eligibility/income_limit.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+description: IRS's Volunteer Income Tax Assistance (VITA) sites offer free tax help to people who make income no more than this amount.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/irs/vita/eligibility/income_limit.yaml:1]:
```suggestion
description: The IRS extends the Volunteer Income Tax Assistance (VITA) program to filers with gross income at or below this amount.
```

File: policyengine_us/variables/gov/irs/vita/vita_eligibility.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class vita_eligibility(Variable):
+    value_type = bool
+    entity = Person
+    label = "IRS Vita program eligibility"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/irs/vita/vita_eligibility.py:7]:
```suggestion
    label = "Eligible for the VITA program"
```

File: policyengine_us/variables/gov/irs/vita/vita_eligibility.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.irs.vita.eligibility
+        income = person("irs_gross_income", period)
+        is_eligible_income = income <= p.income_limit
+        is_disabled = person("is_disabled", period)
+        is_eligible_language = person("is_english_proficient", period)
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/irs/vita/vita_eligibility.py:15]:
```suggestion
        proficient_in_english = person("is_english_proficient", period)
```

File: policyengine_us/variables/gov/irs/vita/vita_eligibility.py [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+        eligible = is_eligible_income | is_disabled | ~is_eligible_language
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/irs/vita/vita_eligibility.py:16]:
```suggestion
        return is_eligible_income | is_disabled | ~proficient_in_english
```

File: policyengine_us/variables/household/demographic/person/is_english_proficient.py [hunk 8]
```diff
@@ -0,0 +1,8 @@
+from policyengine_us.model_api import *
+
+
+class is_(Variable):
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/household/demographic/person/is_english_proficient.py:4]:
```suggestion
class is_english_proficient(Variable):
```

Comment (PavelMakarchuk) [reply policyengine_us/variables/household/demographic/person/is_english_proficient.py:4]: Not sure what happened but this caused the test failure

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/variables/gov/irs/vita/vita_eligibility.py [hunk 9]
```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+
+
+class vita_eligibility(Variable):
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/irs/vita/vita_eligibility.py:4]: and filename
```suggestion
class vita_eligible(Variable):
```

File: policyengine_us/parameters/gov/irs/vita/eligibility/income_limit.yaml [hunk 10]
```diff
@@ -0,0 +1,10 @@
+description: The IRS extends the Volunteer Income Tax Assistance (VITA) program to filers with gross income at or below this amount.
+values:
+  2021-01-01: 64_000
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/irs/vita/eligibility/income_limit.yaml:3]: Source for 2021 start year?

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/irs/vita/eligibility/income_limit.yaml [hunk 11]
```diff
@@ -0,0 +1,10 @@
...
+metadata:
+  unit: currency-USD
+  period: year
+  label: VITA Program Income Limit
+  reference:
+    - title: Free tax return preparation for qualifying taxpayers
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/irs/vita/eligibility/income_limit.yaml:9]: Could we add more references - especially for the 2021 start date

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**